ql/src/java/org/apache/hadoop/hive/ql/exec/repl/RangerDenyTask.java

```diff
@@ -131,6 +141,23 @@ public int execute() {
         }
     }
 
+    private boolean isSetRangerDenyPolicyForReplicatedDb(String rangerEndpoint, String rangerHiveServiceName,
```

Comment:
this can be expensive. Is there any other alternative to exporting all the ranger policies

Reply:
One way can be save this information in dump dir so that next iteration can know ranger deny policy was already created in previous iteration.